conduct.html

```diff
@@ -0,0 +1,114 @@
+    <p>
+      The following are <b>never allowed</b>, regardless of whether you are
+      being sincere or ironic:
+    </p>
+    <ul>
+      <li>Racism</li>
+      <li>Sexism</li>
+      <li>Homophobia</li>
+      <li>Transphobia</li>
+      <li>Harassment</li>
+    </ul>
+    <p>
+      The following specific things, while common in other Eve & gaming
+      contexts, are <b>not allowed</b>:
+    </p>
+    <ul>
+      <li>Use of 'gay' and similar as synonyms for 'uncool'</li>
+      <li>Use of 'jew' and similar as synonyms for 'carebear'</li>
+      <li>Use of 'bitch' and similar for any reason</li>
+      <li>Use of 'rape' and related verbs/terms for any reason</li>
+      <li>Nationalism/racism, either of the 'USA #1!' or 'so-and-so race/ethnic
+group/... are all drunks' type</li>
+    </ul>
+    <p>
+      We also do not allow and <b>will immediately ban you</b> for:
+    </p>
+    <ul>
+      <li>Posting RL information about other players (doxxing)</li>
+      <li>Either selling or buying ISK for real currency (RMT)</li>
+      <li>Automating the Eve client (botting)</li>
+      <li>Out-of-game disruption of another group (DoS or hacking)</li>
+    </ul>
+
+    <p>
+      Deliberate attempts to either "lawyer" (by following the letter but not
+      the spirit) or "edgelord" (by toeing the line) these rules will be dealt
+      with harshly.
+    </p>
+
+    <h2><a name="principles" href="#principles">#</a> Principles</h2>
+    <p>
+      <b>Listening</b>: This is the most important principle. If a corpmate
+      tells you something you're doing is bothering them, listen! Stop the
+      behavior right away <b>even if you think you aren't doing anything
+      wrong</b>. If you need help resolving a disagreement with a corpmate about
+      whether something is acceptable, ask for help in #safe on Slack, or ask
+      any director.
+    </p>
+
+    <p>
+      <b>Trash talk</b>: we do not generally do this, but you are free to if you
+      want. However, this should stay lighthearted and not be mean-spirited;
+      we are playing a video game and the other players are not your real-life
+      enemies.
+    </p>
+
+    <p>
+      <b>Local spam</b>: we generally do not spam in Local (pasting the same
+      message over & over, pasting ASCII art, etc) or in other in-game channels.
+      A coordinated "gf" after an engagement or a "never bend the knee!" after
+      a fight with CVA are fine, though. Use your judgment.
+    </p>
+
+    <p>
+      <b>Drinking/drugs</b>: while it is (unlike in real life) fine to drink
+      and fly or "fly high", you are still expected to abide by this code.
+      Bear in mind that how drunk or high you are is probably a lot less
+      interesting to your corpmates than it seems to you. Please also respect
+      that your alliance mates might not want to hear about it - and if someone
+      asks you to take the discussion elsewhere, do so.
```

Comment:
Since we're talking about principles, there is no need to preface with "while" (see also next point)
```suggestion
      <b>Drinking/drugs</b>: you are expected to abide by this code no matter your intoxication.
      How drunk or high you are is probably a lot less
      interesting to your corpmates than it seems to you. Please also respect
      that your alliance mates might not want to hear about it - and if someone
      asks you to take the discussion elsewhere, do so.
```